test/routable_component.js

```diff
@@ -63,75 +63,75 @@ describe('RoutableComponent', function() {
         );
         this.MyEventSourceComponent.emit('myevent', "1", 2),
         assert(this.subject.controllers.MyController.component === this.subject);
-        assert.deepEqual(this.subject.controllers.MyController.myaction_args, ["1", 2]);
+        assert.deepEqual(this.subject.controllers.MyController.myactionArgs, ["1", 2]);
```

Comment:
__[eslint]__
* Error - Unexpected 'this'. (no-invalid-this)
* Error - Strings must use singlequote. (quotes)
* Warning - No magic number

<!-- Comment by [SideCI](https://sideci.com) -->